localization/es/singleton/README.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Singleton
+category: Creational
+language: en
+tag:
+ - Gang of Four
+---
+
+## Intención
```

Comment:
For this context, I think is better to use _Propósito_ instead of _Intención_, what do you think?

Reply:
Done